### ISSUE TITLE
Script not working any more.

### ISSUE THREAD

**Issue by talzac** [ghwns9652/Minecraft-Bedrock-Server-Updater#3]
The script is not wokring I get this error:
ERROR:root:Timeout raised

And then it install an old version...
root@vmi1582561:/volume1/Minecraft-Bedrock-Server-Updater/updater# python3 mcser                                                                                                                                                             ver_autoupdater_debug.py
ERROR:root:Timeout raised, recovering
Using backup link: https://minecraft.azureedge.net/bin-linux/bedrock-server-1.20                                                                                                                                                             .51.01.zip
Download link: https://minecraft.azureedge.net/bin-linux/bedrock-server-1.20.51.                                                                                                                                                             01.zip
--2024-04-22 09:02:26--  https://minecraft.azureedge.net/bin-linux/bedrock-serve                                                                                                                                                             r-1.20.51.01.zip
Resolving minecraft.azureedge.net (minecraft.azureedge.net)... 152.199.19.161, 2                                                                                                                                                             606:2800:133:206e:1315:22a5:2006:24fd
Connecting to minecraft.azureedge.net (minecraft.azureedge.net)|152.199.19.161|:                                                                                                                                                             443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 121562366 (116M) [application/octet-stream]
Saving to: ‘/volume1/Minecraft-Bedrock-Server-Updater/updater/bedrock-server-1.2                                                                                                                                                             0.51.01.zip’

bedrock-server-1.20 100%[===================>] 115.93M  45.0MB/s    in 2.6s

2024-04-22 09:02:28 (45.0 MB/s) - ‘/volume1/Minecraft-Bedrock-Server-Updater/upd                                                                                                                                                             ater/bedrock-server-1.20.51.01.zip’ saved [121562366/121562366]

ERROR:root:Timeout raised, recoveringserver successfully turned off
backup erased
created new backup
new server unzipped
world migrated
server setting migrated
server turned on
[2024-04-22 09:03:40] minecraft server is updated (v1.20.73.01 -> v1.20.51.01)

**Comment by kristian200sx** [comment]
I got it working modifying the two wget lines in updater/mcserver_autoupdater.py. Seems to be missing the headers.
subprocess.run(['wget', '--user-agent="Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36"', '-P', minecraft_directory+'/updater', '-c', download_link])

**Comment by shaiku** [comment]
That's right, the user agent needs to be set to something real or else the server will reset the connection.  Maybe I'll make a pull request for this fix.
